### PR TITLE
Add volunteering guide blog & 2023 theme (temp fix)

### PR DESCRIPTION
**Pull request by inovizz**

### CODE REVIEW

File: content/blog/2023/volunteering.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+Title: Help us make PyCon India 2023 even more amazing
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:1]: I'm not sure about this.
We've had almost the same title for 2020, and 2021 too 😆 

But I'm fine with it if you are.

File: content/blog/2023/volunteering.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+## What's Expected from Conference Volunteers
+
+All aspects of the conference are managed by volunteers. You can choose to contribute to various workgroups, including:
+
+### Review
+
+- Chair: [Soumendra](https://pyconindia.zulipchat.com/#narrow/pm-with/231777-Soumendra)
+- Shadow: TBA
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/381471-wg.2Freview
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:36]:
```suggestion
- Zulip Stream: <https://pyconindia.zulipchat.com/#narrow/stream/381471-wg.2Freview>
```

File: content/blog/2023/volunteering.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+### Technology
+
+- Chair: [Bharat Saraswat](https://pyconindia.zulipchat.com/#narrow/pm-with/231146-Bharat-Saraswat)
+- Shadow: [Priyanshu Krishnan](https://pyconindia.zulipchat.com/#narrow/pm-with/603863-Priyanshu-Krishnan)
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/226092-wg.2Ftechnology
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:42]:
```suggestion
- Zulip Stream: <https://pyconindia.zulipchat.com/#narrow/stream/226092-wg.2Ftechnology>
```

File: content/blog/2023/volunteering.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+### Platform
+
+- Chair: [Ananyo Maiti](https://pyconindia.zulipchat.com/#narrow/pm-with/231456-Ananya-Maiti-|-EPAM-India)
+- Shadow: [Shashank Kumar](https://pyconindia.zulipchat.com/#narrow/pm-with/234834-Shashank-(realslimshanky)---Volunteer-Manager)
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/226092-wg.2Ftechnology
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:48]:
```suggestion
- Zulip Stream: <https://pyconindia.zulipchat.com/#narrow/stream/226092-wg.2Ftechnology>
```

File: content/blog/2023/volunteering.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+### Promotions/Communications
+
+- Chair: [Bhushan Kapkar](https://pyconindia.zulipchat.com/#narrow/pm-with/608047-BHUSHAN-KAPKAR)
+- Shadow: [Amritha Madam](https://pyconindia.zulipchat.com/#narrow/pm-with/608219-Amritha-Madam)
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/236702-wg.2Fcommunications
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:54]:
```suggestion
- Zulip Stream: <https://pyconindia.zulipchat.com/#narrow/stream/236702-wg.2Fcommunications>
```

File: content/blog/2023/volunteering.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+### Mentorship
+
+- Chair: [Gajendra Deshpande](https://pyconindia.zulipchat.com/#narrow/pm-with/317268-Gajendra-Deshpande)
+- Shadow: TBA
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/245487-wg.2Fmentorship
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:60]:
```suggestion
- Zulip Stream: <https://pyconindia.zulipchat.com/#narrow/stream/245487-wg.2Fmentorship>
```

File: content/blog/2023/volunteering.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+
+### Diversity
+
+- Chair: TBA
+- Shadow: TBA
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/247766-wg.2Fdiversity
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:66]:
```suggestion
- Zulip Stream: <https://pyconindia.zulipchat.com/#narrow/stream/247766-wg.2Fdiversity>
```

File: content/blog/2023/volunteering.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+### Design
+
+- Chair: [Aravind Sekar](https://pyconindia.zulipchat.com/#narrow/pm-with/270428-Aravind-Sekar)
+- Shadow: [Mehboob Subuhani](https://pyconindia.zulipchat.com/#narrow/pm-with/319016-Mehaboob-Subuhani)
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/226098-wg.2Fdesign
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:72]:
```suggestion
- Zulip Stream: <https://pyconindia.zulipchat.com/#narrow/stream/226098-wg.2Fdesign>
```

File: content/blog/2023/volunteering.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+
+### Content
+
+- Chair: [Nikhil Maan](https://pyconindia.zulipchat.com/#narrow/pm-with/231330-Nikhil-Maan)
+- Shadow: Jahnavi J
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/246476-wg.2Fcontent
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:78]:
```suggestion
- Shadow: [Jahnavi J](https://pyconindia.zulipchat.com/#narrow/pm-with/611198-JahnaviJ)
- Zulip Stream: <https://pyconindia.zulipchat.com/#narrow/stream/246476-wg.2Fcontent>
```

File: content/blog/2023/volunteering.md [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+## Weekly Check-in Call Details, feel free to join
+
+- Weekly check-in calls happen every week online on Thursday at 8:00 pm IST.
+- Download the calendar invite [here](https://drive.google.com/file/d/1ZCv0SWPJ0fklxnpnzidX7TK4U4SMbSe_/view?usp=sharing).
+- Join the call via Jitsi using the following link: [Jitsi Link](https://meet.jit.si/pyconindia2023-weekly)
+
+## Additional Volunteering FAQs
+
+1. Where can I contribute?
+    - Review the list of Zulip workstreams for different WorkGroups (WGs) [here](https://pyconindia.zulipchat.com/#narrow/stream/301938-WG-.2F-Team-Names).
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:130]: "This stream does not exist or is private."

File: content/blog/2023/volunteering.md [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+
+4. When should I get started?
+    - Join your interested workgroups and be patient. Workgroups kick off their tasks at different times, so keep an eye out for updates.
+
+5. What is the conference format?
+    - The conference spans four days, including one day of workshops, two days of conference sessions, and a final day dedicated to devsprints.
+
+6. When is the conference?
+    - The conference will take place from September 29 to October 2. Workshops will be held on September 29, conference sessions on September 30 and October 1, and devsprints on October 2.
+
```
Comment (Sc0rpi0n101) [inline content/blog/2023/volunteering.md:146]: We should also add a question about tickets

```suggestion

7.  Do I need to still buy a ticket if I volunteer?
    - Yes. PyCon India is a conference built for the community, by the community. Everyone from the speakers to volunteers, to even the organizers have to pay for their own tickets to attend the conference. Nobody makes a dime during the whole event. All proceedings from the ticket sales go towards making the conference even better for everyone.

    - Here is a beautiful article by [Jesse Noller](https://twitter.com/jessenoller) about the same [http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays](https://in.pycon.org/2020/faq/jessenoller.com/blog/2011/05/25/pycon-everybody-pays).

```

File: pelicanconf.py [hunk 12]
```diff
@@ -21,7 +21,7 @@
 AUTHOR_FEED_RSS = None
 
 ARTICLE_PATHS = ["blog"]
-THEME = "themes/inpycon2021"
+THEME = "themes/inpycon2023"
 
 # Blogroll
 LINKS = (("InPyCon", "https://in.pycon.org/"), ("PSSI", "http://pssi.org.in/"))
```
Comment (Sc0rpi0n101) [inline pelicanconf.py:27]: I'm not sure where these links are used, but if PSSI is not active anymore, we should remove this.

**Review by Sc0rpi0n101 — COMMENTED** [review]
Since when is a comment here required for reviews!?

File: content/blog/2023/volunteering.md (hunk 1 above)
Comment (inovizz) [reply content/blog/2023/volunteering.md:1]: oh is it :D I'll change it.

File: pelicanconf.py (hunk 12 above)
Comment (inovizz) [reply pelicanconf.py:27]: I am not sure, so I left as it is. Let me add a note and we'll remove it when we update the final theme for Blog 2023.

File: content/blog/2023/volunteering.md (hunk 10 above)
Comment (inovizz) [reply content/blog/2023/volunteering.md]: Fixed it. Thanks.

**Comment by inovizz** [comment]
> Since when is a comment here required for reviews!?

@Sc0rpi0n101 - thanks a lot for the review, can you have another look?

**Comment by inovizz** [comment]
Here is the new preview - https://deploy-preview-412--inpyconblog.netlify.app/2023/volunteering-guide-pyconin23.html

**Review by Sc0rpi0n101 — APPROVED** [review]
LGTM

**Pull request merged**